forecasttools/to_flusight.py

```diff
@@ -0,0 +1,96 @@
+from datetime import datetime, timedelta
+
+import polars as pl
+
+
+# NOTE: needs to change
+def calculate_epidate(date):
+    iso_calendar = date.isocalendar()
+    return iso_calendar[1], iso_calendar[0]
+
+
+def get_flusight_target_end_dates(
+    reference_date: str, horizons=None
+) -> pl.DataFrame:
+    # default horizons
+    if horizons is None:
+        horizons = list(range(-1, 4))
+    reference_date_dt = datetime.strptime(reference_date, "%Y-%m-%d")
```

Comment:
Better to expect `reference_date` to be a `datetime` rather than string?

Reply:
This is what I originally had but opted against, since I have the impression that most people would prefer passing the a string than a `datetime` object, hope the function handles converted use adequately. My mind is open to this thought. There are date usage patterns that others are more familiar with than I am.

Reply:
This is fine, but please document the required date format,

Reply:
I still don't see the date format requirement documented.